content/updates/productivity/3-A-more-helpful-process-tracker.md

```diff
@@ -0,0 +1,41 @@
+# More helpful process tracker
+
+## See all the ongoing jobs
+::: meta-data type="Improvement" ee-only="false" link-to-doc="../articles/monitor-jobs.html"
+:::
+
+To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker` to see all jobs of any status under the Process Tracker.
```

Comment:
To give you more visibility and control over all the PIM jobs, we added a new permission: `View all jobs in the Process Tracker`. This permission allows you to see all jobs, regardless of who runs them or what their status is in the process tracker.